web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v340/migrate-default.sql

```diff
@@ -5,5 +5,7 @@ INSERT INTO Settings (name, value, datatype, position, internal) VALUES
 
 INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('system/metadata/validation/removeSchemaLocation', 'false', 2, 9170, 'n');
 
+UPDATE Settings SET name = 'system/localrating/enable', datatype = 0 WHERE id = 211;
```

Comment:
In the UI the values seem `off`, `basic`, `advanced`, I think the migration script should handle to change the previous values `false`, `true`.

Also should be added to the proper migration file, 340 is no longer valid when the PR is merged.